tests/plugins/autocomplete/autocomplete.js

```diff
@@ -97,6 +97,99 @@
 			ac.destroy();
 		},
 
+		'test tab inserts match': function() {
+			var editor = this.editor, bot = this.editorBot, editable = editor.editable(),
```

Comment:
Don't define/declare multiple vars in a single line. Do only one var, per one line - it's a more diff friendly approach.

Reply:
There's no need to store `bot` variable, you're using it only once, and it's very short. The same should be applied to other test cases.